### ISSUE TITLE
Sound bug with back attack combo in JUMP

### ISSUE THREAD

**Issue by Alua7** [MoArtis/RCGMod#37]
Some of "gameplayModifications" section causes jump and heavy attack with Kyoko to do scary sounds.
Update - it seems like it's "backattackcombo" feature.
Upd. video - https://youtu.be/cG6gy8qv3MU
Jump and try left right quickly and heavy attack to opponent

**Comment by Alua7** [comment]
If possible to insert check for jumping status for this feature - perhaps it will heal it?

**Comment by MoArtis** [comment]
yeah I need to look into all of these bugs. But between the virus and my own project time is getting short again. 😁 I can't promise anything but I will try to make up for some time later this month.

**Comment by Alua7** [comment]
Wait a minute, what virus? we need you alive! 
![MikuAA](https://user-images.githubusercontent.com/60073377/73589282-6a0f7000-44fe-11ea-883a-7b9b1cce9927.png)

**Comment by MoArtis** [comment]
@Alua7 as mentioned in my bio, I'm living in Wuhan, where the nCov originated. For real ahah

**Comment by Alua7** [comment]
Why YOU.  T_T (( Why not entire blizzard ent.

**Comment by soulofrebirth** [comment]
> yeah I need to look into all of these bugs. But between the virus and my own project time is getting short again. 😁 I can't promise anything but I will try to make up for some time later this month.

Take care of yourself.
